### ISSUE TITLE
pi not summing to one

### ISSUE THREAD

**Issue by kimkyu80** [GuHongyang/VaDE-pytorch#5]
Hello, thank you for sharing the code.

In the original paper, pi are parameters for categorical variable, so it must sum to one.
However, in this code, when I print out the summation of pi, it exceeds the value one.
Shouldn't there be some kind of softmax or noramlization to prevent this? (SoftMax seems to degrade the performance though...)

**Comment by lileicv** [comment]
I'm confused about that, too.

Have you worked out the problem?
